lib/validation/validators.js

```diff
@@ -85,21 +85,27 @@ module.exports = Validators = {
   },
 
   // eslint-disable-next-line no-inline-comments, spaced-comment
-  before(value/*, [diff, unit][, diff, unit]*/) {
-    const args = [].slice.call(arguments, 1);
-    let date = moment(value, dateFormat);
-    let diff;
-    let unit;
-    while (args.length) {
-      diff = args.shift();
-      unit = args.shift() || 'years';
-      date = date.add(diff, unit);
+  before(value, date/*, [diff, unit][, diff, unit]*/) {
```

Comment:
The commented arguments here are now misleading, I'd remove them and add a separate comment with the `[diff, unit]` method signature.

i.e. `// before(value, [diff, unit][, diff, unit])`